db/seeds/prisons/DMI-durham.yml

```diff
@@ -37,9 +37,9 @@ anomalous:
   2019-11-06:
   - 1715-1915
 unbookable:
-- 2023-07-31
 - 2023-08-17
 - 2023-09-25
+- 2023-09-27
 - 2023-10-30
 - 2023-11-27
-- 2023-12-18
\ No newline at end of file
+- 2023-12-18
```

Comment:
what this one it's on the ticket?

Reply:
added newline at EOF - view inline and you can see it's removed + re-added